apim/Initialize-ApimDeveloperPortal.ps1

```diff
@@ -53,9 +84,18 @@ foreach ($EnvironmentName in $EnvironmentNames) {
     "--uri", "https://management.azure.com/subscriptions/$($AzureSubscriptionId)/resourceGroups/$($ApimResourceGroupName)/providers/Microsoft.ApiManagement/service/$($ApimName)/portalsettings/signup?api-version=2019-12-01",
     "--body", "{'properties': {'enabled': 'false', 'termsOfService': {'enabled': 'false', 'consentRequired': 'false'}}}",
     "--headers", "{'Content-Type':'application/json'}"
-
     az rest @RemoveUsernamePasswordApimIdentityParameters
 
+    # --- Remove default Echo API
+    $RemoveEchoApiParameters =
```

Comment:
Just out of interest, what happens to products that contains only one API. They don't get deleted by default I'm guessing?

Reply:
Believe the API remains if the product is deleted.

https://docs.microsoft.com/en-us/rest/api/apimanagement/2019-12-01/product/delete doesn't work when deleteSubscriptions=true so that's why the product's subscription is deleted separately.